packages/flutter/test/widgets/editable_text_show_on_screen_test.dart

```diff
@@ -233,4 +282,17 @@ void main() {
     expect(scrollController.offset, greaterThan(0.0));
     expect(find.byKey(container), findsNothing);
   });
-}
\ No newline at end of file
+}
+
+class NoImplicitScrollPhysics extends AlwaysScrollableScrollPhysics {
+
+  const NoImplicitScrollPhysics({ ScrollPhysics parent }) : super(parent: parent);
+
+  @override
+  bool get allowImplicitScrolling => false;
+
+  @override
+  NoImplicitScrollPhysics applyTo(ScrollPhysics ancestor) {
+    return new NoImplicitScrollPhysics(parent: buildParent(ancestor));
+  }
+}
```

Comment:
we should have a test that's the original bug: a text field in a tabbed page view right near the edge of the screen, and verify that nothing happens to the horizontal position when it's focused or you type or whatever.

Reply:
Good point. Added.